bytecomp/translmod.ml

```diff
@@ -229,6 +229,8 @@ let undefined_location loc =
                       Const_base(Const_int char)]))
 
 let init_shape modl =
+  let add_name x name =
+    Const_block (0, Lambda.default_tag_info, [x; Const_base (Const_string (name, None))]) in
```

Comment:
It seems we have to change the type, can we guard this using `Clflags.bs_only` and leave the native backend untouched

Reply:
To make the diff minimal, this could be changed to `add_id x (id : Ident.t)`

Reply:
done